proposal/comp110_architecture.bib

```diff
@@ -7,3 +7,91 @@ @article{shannon
 	number={3}, 
 	pages={379-423}
 }
+
+@ARTICLE{smith:launchpad, 
+author={Smith, G. and Whitehead, J. and Mateas, M. and Treanor, M. and March, J. and Mee Cha}, 
+journal={IEEE Transactions on Computational Intelligence and AI in Games}, 
+title={Launchpad: A Rhythm-Based Level Generator for 2-D Platformers}, 
+year={2011}, 
+volume={3}, 
+number={1}, 
+pages={1-16}, 
+month={March},
+}
+
+@ARTICLE{compton:platform, 
+author={Compton, Kate and Mateas, Michael}, 
```

Comment:
This paper seems a bit general. Would, however, make excellent supporting material to help contextualize the comparisons.